StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift

```diff
@@ -255,9 +255,20 @@ final class PaymentSheetAnalyticsHelper {
             guard let elementsSession else { return nil }
             return PaymentSheet.isLinkEnabled(elementsSession: elementsSession, configuration: configuration)
         }()
+        
         var additionalParams = [:] as [String: Any]
         additionalParams["duration"] = duration
         additionalParams["link_enabled"] = linkEnabled
+        if event == .paymentSheetLoadSucceeded {
+            if linkEnabled == true {
+                let linkPassthrough: Bool? = {
+                    guard let elementsSession else { return nil }
+                    return elementsSession.linkPassthroughModeEnabled
+                }()
+                let linkMode: String = linkPassthrough == true ? "passthrough" : "payment_method_mode"
```

Comment:
Should be able to write as 
`let linkMode: String = elementsSession.linkPassthroughModeEnabled ?? false ? "passthrough" : "payment_method_mode"`